main/params.go

```diff
@@ -135,7 +137,10 @@ func init() {
 	bootstrapIDs := fs.String("bootstrap-ids", "default", "Comma separated list of bootstrap peer ids to connect to. Example: NodeID-JR4dVmy6ffUGAKCBDkyCbeZbyHQBeDsET,NodeID-8CrVPQZ4VSqgL8zTdvL14G8HqAfrBr4z")
 
 	// Staking:
-	consensusPort := fs.Uint("staking-port", 9651, "Port of the consensus server")
+	consensusPort := fs.Uint("staking-port", 9651, "External port of the consensus server you can override the local listening port with internal-staking-port for PnP")
+	// this is a bit more complicated, but this becomes an override for port our process will start up listening on.
+	// i did this because I think changing the meaning of 'staking-port' param could be a bit more confusing.
+	internalStakingPort := fs.Uint("internal-staking-port", 9651, "Internal listening consensus port")
```

Comment:
I think we should replace `9651` with `*consensusPort` so that the internal staking port is, by default, the same as the external one